src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        ArrayList<String> arrayList = new ArrayList<>();
```

Comment:
```suggestion
        List<String> arrayList = new ArrayList<>();
```
always use abstract links to variables

Reply:
thank you
done